### DISCUSSION TITLE
Identify which genes are regulated by transcription factors

### DISCUSSION THREAD

**Discussion by FP-10** [GreenleafLab/ArchR#778] [Questions / Documentation]
Hi
Thank you for developing such a useful tool.

As mentioned in the title, is it possible to find genes regulated by specific transcription factors in ArchR?
For example, which genes are regulated by the transcription factor RUNX1?

**Comment by rcorces** [comment]
Such an analysis requires multiple levels of inference and should be taken with a healthy does of skepticism. First, you would infer which TF is binding at a certain peak region based off motif presence in that peak (`addMotifAnnotations()`). Then you would need to infer which gene that peak is regulating (for ex. by `addPeak2GeneLinks()` or something else).
